styles.css

```diff
@@ -125,3 +141,302 @@ body {
   margin-top: 12px;
   margin-bottom: 130px;
 }
+
+.work-container {
+  width: 100vw;
+  height: auto;
+  padding-bottom: 150px;
+  display: grid;
+  grid-template-columns: 327px;
+  justify-content: center;
+  gap: 50px;
+}
+
+.recent-work {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: 0.95fr;
+  align-items: center;
+  padding-top: 120px;
+  background: #fff;
+  top: 682px;
+}
+
+.recent-work h2 {
+  font-size: 38px;
+  font-family: "Crete Round", sans-serif;
+  color: #091e42;
+  font-weight: 400;
+  text-align: center;
+  transition: 0.5s;
+}
+
+hr {
+  width: 100%;
+  border: none;
+  border-top: 1.5px solid #091e42;
+}
+
+.post-stories {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- kindly use the appropriate color for this section as required above by the previous   I suggest you to use this `color: #091E42;` instead of black it will make your design stick to the Figma design as required in the project